src/Tizen.NUI.Components/Controls/Popup.cs

```diff
@@ -71,20 +69,17 @@ public Popup(PopupAttributes attributes) : base(attributes)
         {
             Initialize();
         }
+
         /// <summary>
         /// An event for the button clicked signal which can be used to subscribe or unsubscribe the event handler provided by the user.<br />
         /// </summary>
         /// <since_tizen> 6 </since_tizen>
-        /// This will be public opened in tizen_5.5 after ACR done. Before ACR, need to be hidden as inhouse API.
-        [EditorBrowsable(EditorBrowsableState.Never)]
         public event EventHandler<ButtonClickEventArgs> PopupButtonClickedEvent;
```

Comment:
View has TouchEvent and new Button also has ClickEvent. Let's use the present verb. (PopupButtonClickedEvent->PopupButtonClickEvent)

Reply:
Done